### PR TITLE
Adding in Pagerduty support

### PR DESCRIPTION
**Pull request by nicksteenstra**
## Proposed changes

Adding in new config to support Pagerduty.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ X] I have read the [CONTRIBUTING](https://github.com/bearer/pizzly/blob/master/README.md#contributing-guide) guide
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: integrations/pagerduty.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "authType": "OAUTH2",
+    "tokenParams": { "grant_type": "authorization_code" },
+    "authorizationParams": {},
+    "auth": { "response_type": "code" }
+  },
+  "request": {
+    "baseURL": "https://api.pagerduty.com/",
+    "headers": {
+      "Accept": "application/vnd.pagerduty+json;version=2",
+      "Authorization": "token ${auth.accessToken}",
```
Comment (Frenchcooc) [inline integrations/pagerduty.json:15]:
```suggestion
      "Authorization": "Bearer ${auth.accessToken}",
```
According to: https://developer.pagerduty.com/docs/app-integration-development/oauth-2-auth-code-grant/#using-an-access-token

**Review by Frenchcooc — CHANGES_REQUESTED** [review]
Not sure about the authorization header? Or can you confirm you've tested the configuration first?

File: integrations/pagerduty.json (hunk 1 above)
Comment (nicksteenstra) [reply integrations/pagerduty.json:15]: Yup! I missed that piece. Definitely needs to be Bearer there.

**Review by Frenchcooc — APPROVED** [review]
LGTM 🎉

**Pull request merged**